roachtest: pgx failed

roachtest.pgx [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3344851&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3344851&tab=artifacts#/pgx) on master @ [46cef2c6f0b36ba2f7d551c8ab017832c1b9d592](https://github.com/cockroachdb/cockroach/commits/46cef2c6f0b36ba2f7d551c8ab017832c1b9d592):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgx/run_1
	orm_helpers.go:245,orm_helpers.go:171,pgx.go:123,pgx.go:134,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-3544-g46cef2c6f0
		Tests run against pgx v4.11.0
		185 Total Tests Run
		184 tests passed
		1 test failed
		29 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestConnCopyFromFailServerSideMidwayAbortsWithoutWaiting - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blocklist (pgxBlocklist21_2) is available in the artifacts' pgx log
```
<details><summary>Reproduce</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/roachtest)



See: [CI job to stress roachtests](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestStress)

<p>For the CI stress job, click the ellipsis (...) next to the Run button and fill in:
* Changes / Build branch: master
* Parameters / `env.TESTS`: `^pgx$`
* Parameters / `env.COUNT`: &lt;number of runs&gt;
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #65041 roachtest: pgx failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-20.1 release-blocker]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*pgx.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.pgx [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3352195&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3352195&tab=artifacts#/pgx) on master @ [967ed00f80981ce8848a5e8144ee6fbd29bc95bb](https://github.com/cockroachdb/cockroach/commits/967ed00f80981ce8848a5e8144ee6fbd29bc95bb):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgx/run_1
	orm_helpers.go:245,orm_helpers.go:171,pgx.go:123,pgx.go:134,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-3579-g967ed00f80
		Tests run against pgx v4.11.0
		185 Total Tests Run
		184 tests passed
		1 test failed
		29 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestConnCopyFromFailServerSideMidwayAbortsWithoutWaiting - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blocklist (pgxBlocklist21_2) is available in the artifacts' pgx log
```
<details><summary>Reproduce</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #65041 roachtest: pgx failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-20.1 release-blocker]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*pgx.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/pgx_blocklist.go
+++ b/pkg/cmd/roachtest/tests/pgx_blocklist.go
@@ -30,6 +30,7 @@ var pgxIgnorelist21_2 = pgxIgnorelist21_1
 var pgxIgnorelist21_1 = pgxIgnorelist20_2
 
 var pgxIgnorelist20_2 = blocklist{
-	"v4.TestBeginIsoLevels":   "We don't support isolation levels",
-	"v4.TestQueryEncodeError": "This test checks the exact error message",
+	"v4.TestBeginIsoLevels": "We don't support isolation levels",
+	"v4.TestConnCopyFromFailServerSideMidwayAbortsWithoutWaiting": "https://github.com/cockroachdb/cockroach/issues/69291#issuecomment-906898940",
+	"v4.TestQueryEncodeError":                                     "This test checks the exact error message",
 }

